File: PolarMathCS/PolarMath/Evaluation/Spectrum/VLFPercents.cs
using PolarMath.Data;
using System;

namespace PolarMath.Evaluation.Spectrum
{
    public class VLFPercents : IEvaluation<double>
    {
        public Double Evaluate(SessionData training)
        {
            var vlf = training.Evaluate( new VLF() );
            var tp = training.Evaluate( new TP() );
            return (vlf / tp) * 100;
        }
    }
}

File: PolarMath/src/org/macondo/polar/evaluation/spectrum/VLFPercents.java
package org.macondo.polar.evaluation.spectrum;

import org.macondo.polar.data.Training;
import org.macondo.polar.evaluation.Evaluation;

public class VLFPercents implements Evaluation<Double> {
	
	public Double evaluate(Training training) {
		double vlf = training.evaluate(new VLF());
		double tp = training.evaluate(new TP());
		return (vlf / tp) * 100;
	}

}
